Apps/APN/manager/CioManager.js

```diff
@@ -0,0 +1,34 @@
+import {CustomerIO} from 'customerio-reactnative'
+
+class CioManager {
```

Comment:
Because our sample app's source code is designed to be read by customers to learn how to use our SDK, I suggest that we do not put CIO SDK calls into a separate class. I think that by trying to abstract the CIO SDK into a separate class, we are adding unneeded complexity to the code base making the code harder for readers to read and understand.

Reply:
IMO, having all methods at one place will help understanding the implementation/usage better. Keeping the implementation code in different files based on feature might add unneeded confusion.